monitors/SyntheticPegMonitor.js

```diff
@@ -112,8 +125,10 @@ class SyntheticPegMonitor {
   }
 
   // Checks difference between minimum and maximum historical price over `volatilityWindow` amount of time.
-  // Fires a message if the difference exceeds the `volatilityAlertThreshold` %.
+  // Fires a message if the difference exceeds the `volatilityAlertThreshold` %. `checkPegVolatility` checks if the
+  // reference medianizer price feed has a large % change over the window.
   async checkPegVolatility() {
+    if (this.pegVolatilityAlertThreshold == 0) return; // Exit early if not monitoring peg volatility.
```

Comment:
Can you please use three "=" signs throughout this PR just to stay consistent with our existing style?